libs/dataset-explorer/src/lib/ChartView/utils/DatasetExplorerTab.styles.ts

```diff
@@ -153,7 +153,7 @@ export const datasetExplorerTabStyles: () => IProcessedStyleSet<IDatasetExplorer
       page: {
         color: theme.semanticColors.bodyText,
         height: "100%",
-        padding: "0 40px 10px 40px",
+        padding: "0 40px 32px 40px",
```

Comment:
Shall we move these to some constant strings?

Reply:
It's not uniform everywhere. I applied the same for causality and it didn't make sense there because the component is comprised of different pieces that have their own spacing. I don't want to overthink this one and keep it as a simple fix. If we get time to rethink how these are structured we can do something more holistic and at that time, I agree, this should be done with constants and uniformly.

Reply:
Other than causality, data balance and data explorer (tabular view, but not chart view) were also examples where it didn't make sense to increase.